### DISCUSSION TITLE
How to solve this issues

### DISCUSSION THREAD

**Discussion by HarishankarVj12** [Fantom-foundation/go-opera#464] [General] [answered]
Fatal: Failed to make engine: failed to open existing databases: dirty state: gossip: 0017642C2E50CB4C79,
gossip-213964: DE0017642C2BB99A9AE517642C2E50CB4C79

**Comment by quan8** [comment]
Hi @HarishankarVj12 , do you have Reproduction steps? which opera version and which command did you use?

**Reply by HarishankarVj12** [reply, marked as answer]
Could you please share the command to check the version

**Reply by HarishankarVj12** [reply]
Version: 1.1.1-rc.2

**Reply by HarishankarVj12** [reply]
Now I have updated 1.1.1-rc.5
